drivers/i2c/i2c_cc13xx_cc26xx.c

```diff
@@ -29,7 +29,7 @@ struct i2c_cc13xx_cc26xx_data {
 	struct k_sem lock;
 	struct k_sem complete;
 	volatile uint32_t error;
-#ifdef CONFIG_PM
+#ifdef CONFIG_PM_DEVICE
```

Comment:
this fields are being accessed in places guarded by `CONFIG_PM`

Reply:
Thanks! fixed.